plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesExt.kt

```diff
@@ -219,7 +219,9 @@ fun MapPluginExtensionsDelegate.setGesturesManager(
 }
 
 /**
- * The gesture configuration object.
+ * Get copy of current gesture settings.
+ * This copy should not be used to update any setting, please [updateGestureSettings] instead.
```

Comment:
What happens if settings are updated with this ext?

Reply:
Nothing as you update a copy and not an actual object